### PR TITLE
Median sorted array

### LINKED ISSUE #2018: Median of Two sorted Arrays
Median of two sorted arrays can be calculated using two pointer approach , without merging two arrays in O(n)

### PR DESCRIPTION
**Pull request by ashishnagpal2498**
Fixes #2018 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- C++

#### Files Added:
-  median_ofTwoSortedArrays

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp [hunk 1]
```diff
@@ -0,0 +1,70 @@
+//median of two sorted arrays -
```
Comment (jainaman224) [inline Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp]: Fix file and folder name.

File: Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+#include<iostream>
+using namespace std;
+float median(int *arr1, int * arr2, int n, int m)
```
Comment (jainaman224) [inline Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp]: Add new line above function and blocks.

File: Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp (hunk 1 above)
Comment (ashishnagpal2498) [reply Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp]: Okay , what should be a suitable name for this ?

File: Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp (hunk 2 above)
Comment (ashishnagpal2498) [reply Median_of_two_sorted_Array/median_ofTwoSortedArrays.cpp]: Okay , working on it.

**Comment by SakshiShreya** [comment]
Please squash all your commits.

**Comment by somya-kapoor** [comment]
@ashishnagpal2498 Please remove unwanted files.

**Pull request closed without merging**